tokio/src/runtime/time/mod.rs

```diff
@@ -93,9 +94,6 @@ struct Inner {
     // The state is split like this so `Handle` can access `is_shutdown` without locking the mutex
     pub(super) state: Mutex<InnerState>,
 
-    /// True if the driver is being shutdown.
-    pub(super) is_shutdown: AtomicBool,
```

Comment:
I don't remember this being an atomic, and I assumed it wasn't when I made the comment that the time driver has a similar issue.

Reply:
That makes sense. It's questionable that this change improves performance at all. TBH I have no idea how to benchmark timers only.